git/git.md

```diff
@@ -1,12 +1,12 @@
 # Git
 
-- [Repositories](#repositories)
+- [Repository names](#repositories)
```

Comment:
as this is linking to the ID of the H2 that's created - I think it needs to be

`[Repository names](#repositories-names)`